test/functional/git_controller_test.rb

```diff
@@ -233,22 +233,22 @@ def setup
   end
 
   test 'getting blob with no permissions throws error' do
-    logout
-    get :blob, params: { workflow_id: @workflow.id, version: @git_version.version, path: 'diagram.png' }, format: :html
+    workflow = FactoryBot.create(:local_git_workflow, policy: FactoryBot.create(:publicly_viewable_policy))
+    get :blob, params: { workflow_id: workflow.id, version: 1, path: 'diagram.png' }, format: :html
```

Comment:
these tests ( including the next few below) could also check the response code is `:forbidden`